lib/l10n/app_ar.arb

```
@@ -264,6 +264,62 @@
+  "videoRecordedDuration": "تم تسجيل الفيديو ({duration} ثانية)",
+  "failedToStopRecording": "فشل إيقاف التسجيل: {error}",
+  "galleryError": "خطأ في المعرض",
+  "failedToSelectVideo": "فشل اختيار الفيديو من المعرض: {error}",
+  "preparingVideo": "جارٍ تجهيز الفيديو...",
+  "creatingVideoThumbnail": "جارٍ إنشاء صورة مصغرة للفيديو...",
+  "videoReadyForUpload": "الفيديو جاهز للرفع!",
+  "videoReady": "الفيديو جاهز",
+  "selectedVideoClipReady": "مقطع الفيديو المحدد ({duration}) جاهز للرفع",
+  "processingError": "خطأ في المعالجة",
+  "failedToProcessVideo": "فشل معالجة الفيديو المحدد: {error}",
+  "error": "خطأ",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)